sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -176,6 +176,13 @@ def __init__(self, packages, options, environment_version, pipeline_url):
     if self.debug_options.experiments:
       for experiment in self.debug_options.experiments:
         self.proto.experiments.append(experiment)
+    # Add MULTIPLE_SDK_CONTAINERS flag if its not already present. Do not add
+    # the flag if 'NO_MULTIPLE_SDK_CONTAINERS' is present.
+    # TODO: Cleanup MULTIPLE_SDK_CONTAINERS once we depricate Python SDK till
+    # version 2.4.
+    if ('MULTIPLE_SDK_CONTAINERS' not in self.proto.experiments and
```

Comment:
How confident are we to make this a default behavior for 2.5?

Reply:
I expect this CL to get in 2.5.
In a way this flag is required to help router distinguish between old SDK (sdk till 2.4) and new SDK (sdk from 2.5). So once we do not have any sdk which is older than 2.5, we don't need to distinguish between sdk atleast for MultiSdk functionality and hence it automatically becomes the default feature.

Reply:
@robertwb @aaltay I am planning to make this feature opt out for new SDKs. Instead should we keep it opt in?